api/v1/lib/scheduler/scheduler.proto

```diff
@@ -239,7 +239,8 @@ message Response {
     repeated OperationStatus operation_statuses = 1 [(gogoproto.nullable) = false];
   }
 
-  optional ReconcileOperations reconcile_operations = 1;
+  optional Type type = 1 [(gogoproto.nullable) = false];
```

Comment:
this tag change is unfortunate but aligns perfectly w/ upstream changes

Reply:
I checked with Gaston and confirmed that this was not used anywhere when introduced, so it should be a "safe" change.